Simulation.cc

```diff
@@ -101,10 +118,10 @@ void setupTrackByToyMC(SVector3& pos, SVector3& mom, SMatrixSym66& covtrk, HitVe
     // can I do what I did, generate one (theta_space) and rotate it by another random numbers
     const float z1 = g_gaus(g_gen);
     const float z2 = g_gaus(g_gen);
-    const float phismear = g_unif(g_gen)*TMath::TwoPi(); // random rotation of scattering plane
+    const float phismear = g_unif(g_gen)*Config::TwoPI; // random rotation of scattering plane
     const float X0 = 9.370; // cm, from http://pdg.lbl.gov/2014/AtomicNuclearProperties/HTML/silicon_Si.html
     //const float X0 = 0.5612; // cm, for Pb
-    float x = 0.1; // cm  -assumes radial impact. This is bigger than what we have in main
+    float x = 0.0025; //.1 cm  -assumes radial impact. This is bigger than what we have in main
```

Comment:
@cerati I believe it is related to the scattering somehow.  I must have changed it when I was messing with the beamspot studies.  Seeing as Mplex currently does not use scattering, I just changed it back to 0.1.  

In reality, float x = std::sqrt(getRad2(Config::beamspotX, Config::beamspotY)) -- I think, need to confirm with @pwittich 

And yes, I think x0 should also be moved to Config.h.  I will make a commit for that.